Doc/library/sys.rst

```diff
@@ -814,6 +814,11 @@ always available.
    ``cache_tag`` is set to ``None``, it indicates that module caching should
    be disabled.
 
+   *opt_levels* is a tuple containing the available compiler-supported
+   optimization levels.  See :pep:`488` for more information.
```

Comment:
I think that mentioning the possible values, and the relation to the "optimization" parameter of `importlib.util.cache_from_source()`, is perhaps warranted here (in addition to linking to PEP-488).

Reply:
+1

The relationship between `sys.flags.optimize` and that "optimization" parameter (as well as the "optimize" parameter of `compile()`) reflects a gap in the solution presented by PEP 488.  That needs to be resolved.